e2e-tests/k8s-e2e-bootstraping.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env sh
 
-E2E_KUBERNETES_VERSION=${E2E_KUBERNETES_VERSION:-v1.22.0}
+E2E_KUBERNETES_VERSION=${E2E_KUBERNETES_VERSION:-v1.26.8}
```

Comment:
@xqi-nr is there any reason why not use the latest supported Kubernetes version here (1.27.5)?

Reply:
no, I made it to the latest version. Thanks for pointing it out